ledger-service/http-json/src/main/scala/com/digitalasset/http/Config.scala

```diff
@@ -102,15 +108,24 @@ private[http] final case class JdbcConfig(
     user: String,
     password: String,
     createSchema: Boolean = false,
+    minIdle: Int = JdbcConfig.MinIdle,
```

Comment:
so this part is different from the strict backport right? In the future, I’d recommend separating the backport from additional changes to ease review